### PR TITLE
TSB4: handling codepage and correct text extract

### PR DESCRIPTION
**Pull request by kamenf**

### CODE REVIEW

File: libse/SubtitleFormats/TSB4.cs [hunk 1]
```diff
@@ -89,6 +89,7 @@ public override void LoadSubtitle(Subtitle subtitle, List<string> lines, string
                     string text = Encoding.GetEncoding(CodePage).GetString(array, textStart, length);
                     // text = Encoding.Default.GetString(array, i + 53, endOfText - 47);
                     text = text.Trim('\0').Replace("\0", " ").Trim();
+                    text = text.Replace("\r\n", "\n").Replace("\r", "\n").Replace("\n", "\r\n"); //conform to CRLF
```
Comment (ivandrofly) [inline libse/SubtitleFormats/TSB4.cs]: Perhaps: `string.join(Environment.Newlin, text.SplitToLines());`

Comment (kamenf) [inline libse/SubtitleFormats/TSB4.cs]: Sure. Changing it.

**Comment by niksedk** [comment]
@kamenf: thx for the pullrequest :)
I've two older sample files that do not read too well with latest changes - perhaps we should exchange sample files per email?

**Comment by kamenf** [comment]
@niksedk: Sent to your provided here e-mail :)

**Pull request merged**